core/tools.js

```diff
@@ -1844,6 +1844,39 @@
 				}
 				return acc;
 			}
+		},
+
+		/**
+		 * A set of object helpers.
+		 *
+		 * @property {CKEDITOR.tools.object}
+		 * @member CKEDITOR.tools
+		 */
+		object: {
+			/**
+			 * Return a first key from `obj` which has given value. If key cannot be find, `null` is returned.
+			 *
+			 * @param {Object} obj Object which key is looked for.
+			 * @param {Mixed} value Object's vlaue which key is looked for
```

Comment:
Missing dot, typo in "vlaue". Also I think it would be more readable to put it like "Object's value to be looked for.".